src/query/query_list.h

```diff
@@ -173,7 +173,8 @@ typedef struct xasl_id XASL_ID;
 struct xasl_id
 {
   SHA1Hash sha1;		/* SHA-1 hash generated from query string. */
-  volatile INT32 cache_flag;	/* Multiple-purpose field used to handle XASL cache. */
+  /*volatile */ INT32 cache_flag;
```

Comment:
@vipalade
Do you want to remove the volatile? It doesn't cause compile errors (anymore?).